python/revng/internal/cli/_commands/generate_migrations/metaschema.yml

```diff
@@ -98,26 +98,30 @@ definitions:
       - "$ref": "#/definitions/SequenceStructField"
       - "$ref": "#/definitions/ReferenceStructField"
 
-  SimpleStructField:
-    type: object
-    additionalProperties: false
-    properties:
-      name:
-        type: string
-      type:
-        type: string
-      doc:
-        type: string
-      is_global_index:
-        type: boolean
-        default: false
-      const:
-        type: boolean
-        default: false
-    required:
-      - name
-      - type
-      - doc
+  # DANGER!!!!!
+  # This commit doesn't break anything, how was metaschema supposed to be used?!!
+  # DANGER!!!!!
```

Comment:
@mrjackv: @Cvelth says that commenting random stuff in here has no effect.
Can you comment on this?